frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -73,20 +96,51 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
               breadcrumb={
                 <Breadcrumb>
                   {breadcrumbPath}
-                  <BreadcrumbItem isActive>{pipeline?.name || 'Loading...'}</BreadcrumbItem>
+                  <BreadcrumbItem>{pipeline?.name || 'Loading...'}</BreadcrumbItem>
+                  <BreadcrumbItem isActive>{pipelineVersion?.name || 'Loading...'}</BreadcrumbItem>
                 </Breadcrumb>
               }
-              title={pipeline?.name || 'Loading...'}
-              description={
-                pipeline ? <MarkdownView conciseDisplay markdown={pipeline.description} /> : ''
-              }
+              title={pipelineVersion?.name || 'Loading...'}
+              {...(pipeline && {
+                description: (
+                  <MarkdownView conciseDisplay markdown={pipelineVersion?.description} />
+                ),
+              })}
```

Comment:
```suggestion
              {...(pipelineVersion && {
                description: (
                  <MarkdownView conciseDisplay markdown={pipelineVersion.description} />
                ),
              })}
```

Reply:
Yeah, this is a consequence of too much back and forth about what we want to display and making quick changes. Will update.